playbooks/common/openshift-cluster/upgrades/v3_5/storage_upgrade.yml

```diff
@@ -0,0 +1,19 @@
+---
+###############################################################################
+# Post upgrade - Upgrade job storage
+###############################################################################
+- name: Upgrade job storage
+  hosts: oo_first_master
+  roles:
+  - { role: openshift_cli }
+  vars:
+    # Another spot where we assume docker is running and do not want to accidentally trigger an unsafe
+    # restart.
+    skip_docker_role: True
+  tasks:
+  - name: Upgrade job storage
+    command: >
+      {{ openshift.common.client_binary }} adm --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+      migrate storage --include=jobs --confirm
+    run_once: true
```

Comment:
We need a conditional here to keep it from executing on 3.4 upgrades. Technically we say that master is only 3.5 but as devs and currently in ops we have to use it for 3.4 upgrades, in which case I suspect we don't want this to run.

Example: when: openshift.common.version_gte_3_5_or_1_5

Also if it *shouldn't* be run on 3.6, we need to make that conditional somehow as well. Is this something that should be run on every upgrade going forward?